bridge-school/company-cert-frontend
# Fetch a single company and save in store

Aaand just like that, there is another PR...

Fetching a single company and saving it in redux - let's do all the styling in a separate task.

- connected the `Company` component to store
- created an action to fetch the company data (notice the path for the fetch - it start's with `/`, this makes the path absolute. I had to change this for the `frontend-data` endpoint as well otherwise the app was getting troubles recognizing where to fetch the data from)
- saved the company data in the store

How to test this?
Run the server from `fix/get-single-company` branch (if this branch is already merged then you can use `develop`).
Start the front end and submit a new company - after redirect, you should be able to see the company's name and in the Redux dev tool, you should see the company stored in the store.
